src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.files.ReaderCsvFile;
+import core.basesyntax.files.WriterCsvFile;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.TransactionParser;
+import core.basesyntax.service.impl.FruitShopServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.TransactionParserImpl;
+import core.basesyntax.strategy.BalanceHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.PurchaseHandler;
+import core.basesyntax.strategy.ReturnHandler;
+import core.basesyntax.strategy.SupplyHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT_FILE_NAME = "src/main/resources/inputFile.csv";
+    private static final String OUTPUT_FILE_NAME = "src/main/resources/outputFile.csv";
+
+    private static final Map<Operation, OperationHandler> operationHandlerMap = new HashMap<>();
+    private static final ReaderCsvFile readerCsvFile = new ReaderCsvFile();
+    private static final WriterCsvFile writerCsvFile = new WriterCsvFile();
```

Comment:
```suggestion
    private static final FileReader csvFileReader = new CsvFileReader();
    private static final FileWriter csvFileWriter = new CsvFileWriter();
```
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/665b9fbc-a4c2-47bd-af3f-0c736707ead3)
See **D** - Dependency Inversion principle. Use interface when declaring field.